composer.json

```diff
@@ -29,9 +29,6 @@
         "ext-zip": "Needed to support extraction of ZIP archives when doing downloads or updates"
     },
     "config": {
-        "platform": {
-            "php": "5.6"
-        },
```

Comment:
Removing this will case usage3 of versions of independences without php 6.6 support if `composer install` or `composer update` is run on higher php version.

Reply:
This is only a library, so that's fine. We'll need to fix the versions for the bundle when building a Phar. When using Composer, we don't need to enforce a platform.

Reply:
The only thing that really needs this 5.6 platform requirement is the Phar version of WP-CLI.

Reply:
In CI yes, but what with local development?

Reply:
For local development, you just run `composer install` and you'll get whatever versions are the best match for the version of PHP you're using.

Reply:
ok